drivers/sensor/scd4x/scd4x.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2022 Stephen Oliver
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */   
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_SCD4X_SCD4X_H_
+#define ZEPHYR_DRIVERS_SENSOR_SCD4X_SCD4X_H_
+
+#include <zephyr/device.h>
+
+
+#define SCD4X_MAX_AMBIENT_PRESSURE UINT16_MAX
+
+#define SCD4X_CMD_POWER_DOWN 	0x36E0
+#define SCD4X_CMD_WAKE_UP		0x36F6
+#define SCD4X_CMD_REINIT		0x3646
+
+#define SCD4X_CMD_START_PERIODIC_MEASUREMENT 			0x21B1
+#define SCD4X_CMD_STOP_PERIODIC_MEASUREMENT 			0x3F86
+#define SCD4X_CMD_START_LOW_POWER_PERIODIC_MEASUREMENT 	0x21AC
+
+#define SCD4X_CMD_GET_DATA_READY_STATUS 0xE4B8
+#define SCD4X_CMD_READ_MEASUREMENT 		0xEC05
+
+#define SCD4X_CMD_PERSIST_SETTINGS 	0x3615
+
+#define SCD4X_CMD_GET_SERIAL_NUMBER 0x3682
+
+#define SCD4X_CMD_PERFORM_SELF_TEST 0x3639
+#define SCD4X_CMD_PERFORM_FACTORY_RESET 0x3632
+
+#define SCD4X_CMD_SET_TEMPERATURE_OFFSET 0x241D
+#define SCD4X_CMD_GET_TEMPERATURE_OFFSET 0x2318
+
+#define SCD4X_CMD_SET_SENSOR_ALTITUDE 0x2427
+#define SCD4X_CMD_GET_SENSOR_ALTITUDE 0x2322
+
+#define SCD4X_CMD_SET_AMBIENT_PRESSURE 0xE000
+
+#define SCD4X_CMD_PERFORM_FORCED_RECALIBRATION 0x362F
+
+#define SCD4X_CMD_SET_AUTOMATIC_SELF_CALIBRATION_ENABLED 0x2416
+#define SCD4X_CMD_GET_AUTOMATIC_SELF_CALIBRATION_ENABLED 0x2313
+
+#define SCD4X_POWER_DOWN_WAIT_MS	1
+#define SCD4X_WAKE_UP_WAIT_MS	20
+#define SCD4X_REINIT_WAIT_MS	20
+#define SCD4X_PERFORM_SELF_TEST_WAIT_MS 10000
+#define SCD4X_PERFORM_FACTORY_RESET_WAIT_MS 1200
+#define SCD4X_STOP_PERIODIC_MEASUREMENT_WAIT_MS	500
+#define SCD4X_READ_MEASUREMENT_WAIT_MS	1
+#define SCD4X_SET_TEMPERATURE_OFFSET_WAIT_MS	1
+#define SCD4X_GET_TEMPERATURE_OFFSET_WAIT_MS	1
+#define SCD4X_SET_SENSOR_ALTITUDE_WAIT_MS	1
+#define SCD4X_GET_SENSOR_ALTITUDE_WAIT_MS	1
+#define SCD4X_SET_AMBIENT_PRESSURE_WAIT_MS	1
+#define SCD4X_SET_AUTOMATIC_CALIBRATION_WAIT_MS	1
```

Comment:
I'd probably avoid aligning stuff like these, its just too much effort & not required by zephyr's style. Over time, these alignments tend to rot, since it is not required and developers do things differently